dockerfiles/scripts/install_common_deps.sh

```diff
@@ -21,6 +21,6 @@ pip install "wheel>=0.35.1"
 rm -rf /opt/miniconda/pkgs
 
 # Dependencies: cmake
-wget --quiet https://github.com/Kitware/CMake/releases/download/v3.26.3/cmake-3.26.3-linux-x86_64.tar.gz
-tar zxf cmake-3.26.3-linux-x86_64.tar.gz
```

Comment:
There's timeout issue in 3.27.0.
Does 3.27.1 fix it?

Reply:
It looks the regression isn't fixed.
https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=334111&view=logs&jobId=f9f5b320-fa10-56c4-debe-61ea69c74793&j=f9f5b320-fa10-56c4-debe-61ea69c74793&t=73689a31-9570-57c6-acc6-0f29b528d037

Reply:
It looks the regression isn't fixed.
https://aiinfra.visualstudio.com/Lotus/_build/results?buildId=334111&view=logs&jobId=f9f5b320-fa10-56c4-debe-61ea69c74793&j=f9f5b320-fa10-56c4-debe-61ea69c74793&t=73689a31-9570-57c6-acc6-0f29b528d037

Reply:
3.27.2 still has the issue.

Reply:
I added a timeout value to each test target, then it is fine.